vql/filesystem/zip.go

```diff
@@ -358,9 +358,10 @@ func (self *ZipFileCache) Close() {
 // root scope. We keep a list of most recently used cached of zip
 // files for quick access.
 type ZipFileSystemAccessor struct {
-	mu       sync.Mutex
-	fd_cache map[string]*ZipFileCache
-	scope    vfilter.Scope
+	mu         sync.Mutex
+	fd_cache   map[string]*ZipFileCache
+	scope      vfilter.Scope
+	sourceFile string
```

Comment:
Can you please explain why we need this?  Since the zip accessor usually calls a delegate it probably does not need to specially handle the remapping?

Reply:
I will revisit what I did here. Seeing my changes right now, it appears to me that I had something in mind regarding the pathspec which was conceptually wrong.